src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,21 +1,27 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends defaultCountingOutRhymer {
-
-	public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
-	
-	@Override
-	public int countOut() {
-		while (!callCheck())
-			
-		temp.countIn(super.countOut());
-		
-		int ret = temp.countOut();
-		
-		while (!temp.callCheck())
-			
-		countIn(temp.countOut());
-		
-		return ret;
-	}
+/**
+ * FIFO based implementation of Rhymer
+ */
+public class FIFORhymer extends DefaultCountingOutRhymer {
+
+    private final DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+
+    /**
+     * @return method returns element from queue
+     */
+    @Override
+    public int countOut() {
+        while (!callCheck())
+
+            temp.countIn(super.countOut());
+
+        int ret = temp.countOut();
+
+        while (!temp.callCheck())
+
+            countIn(temp.countOut());
+
+        return ret;
+    }
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.